packages/material-components-web/index.js

```diff
@@ -14,29 +14,29 @@
  * limitations under the License.
  */
 
-import autoInit from '@material/auto-init';
```

Comment:
This looks like a broken change in this PR

Reply:
This is from my merge with master.